Ensure `connects_to` can only be called on base or abstract classes (#40219)
rails/rails: activerecord/CHANGELOG.md

> *   `connects_to` can only be called on `ActiveRecord::Base` or abstract classes

Reviewer: ```suggestion
*   `connects_to` can only be called on `ActiveRecord::Base` or abstract classes.
```

>     Ensure that `connects_to` cannot be called from classes that are not `ActiveRecord::Base` or abstract classes. This protects the application from opening duplicate or too many connections.
> 

Reviewer: ```suggestion
Ensure that `connects_to` can only be called from `ActiveRecord::Base` or abstract classes. This protects the application from opening duplicate or too many connections.
```
I think the PR title to be much better that negating i.e. `cannot that are not`
